app/lib/mavis_cli/vaccination_records/create_from_fhir.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module MavisCLI
+  module VaccinationRecords
+    class CreateFromFHIR < Dry::CLI::Command
+      desc "Create a vaccination record from a FHIR Immunization JSON"
+
+      argument :patient_id,
+               required: true,
+               desc: "ID of the patient to attach the vaccination record to"
+      argument :team_id,
+               required: false,
+               desc: "ID of the team (optional, used for batch association)"
+
+      option :file,
+             type: :string,
+             required: false,
+             desc: "Path to a file containing the FHIR Immunization JSON"
+      option :json,
+             type: :string,
+             required: false,
+             desc: "Raw FHIR Immunization JSON string"
+
+      def call(patient_id:, team_id: nil, file: nil, json: nil, **)
+        MavisCLI.load_rails
+
+        if file.nil? && json.nil?
```

Comment:
```suggestion
        if file.blank? && json.blank?
```

To handle if the user provides an empty string.